widgets/src/command_text_input.rs

```diff
@@ -0,0 +1,610 @@
+use crate::*;
+
+live_design! {
+    link widgets;
+    use link::widgets::*;
+    use link::theme::*;
+
+    List = {{List}} {
+        flow: Down,
+        width: Fill,
+        height: Fill,
+    }
```

Comment:
Brought a simplified copy of the `List` widget from Moly, as `PromptInput` (the source of this widget) was implemented on top of it.

Is it possible to use `FlatList` instead, but it may leak memory. And `PortalList` hasn't been adapted to support `FlatList` use cases.